docs/WindowsBuild.md

```diff
@@ -51,28 +51,31 @@ git clone https://github.com/curl/curl.git
 git clone https://gitlab.gnome.org/GNOME/libxml2.git
 ```
 
-## 3. Acquire ICU
-1. Download ICU from [ICU Project](http://site.icu-project.org) for Windows x64 and extract the folder to a new folder called `thirdparty`. In other words, there should be a folder `S:\thirdparty\icu4c-64_2-Win64-MSVC2017` with the ICU. 
-1. Add the `bin64` folder to your `Path` environment variable.
+## 3. Acquire ICU, SQLite3, curl, libxml2
+1. Go to https://dev.azure.com/compnerd/windows-swift and scroll down to "Dependencies" where you'll see bots (hopefully green) for icu, SQLite, curl, and libxml2. Download each of the zip files and copy their contents into s:/hirdparty. The directory structure should resemble:
+
+```
+s:/thirdparty/
```

Comment:
Capitalise `S` please.  Now is a good time to move this to `S:/Library` as they are no longer built by the projects themselves.